indico/modules/events/registration/templates/display/regforms_list.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'layout/event_page_base.html' %}
+
+{% block title %}
+    {%- trans %}Registration Forms{% endtrans -%}
+{% endblock %}
+
+{% block content %}
+    <table id="survey-list" class="i-table-widget">
+        <thead>
+            <tr>
+                <th>{% trans %}Registration Form{% endtrans %}</th>
+                <th>{% trans %}Available from{% endtrans %}</th>
+                <th>{% trans %}Deadline{% endtrans %}</th>
+                <th></th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for regform in regforms %}
+                <tr>
+                    <td>
+                        {% if event.canModify(session.user) %}
```

Comment:
once #2055 is merged (probably going to do it on monday, want to add some unit tests for critical areas such as access checks first) this would be `event.as_event.can_manage(session.user, 'registration')`